index.html

```diff
@@ -35,38 +31,151 @@ <h1 class="greeting-hearder">I’m Sive <br> Glad to see you!</h1>
         <a href="#" class="connect"><h4>LET'S CONNECT</h4></a>
 
         <ul id="social-icons">
-            <li class="twitter">
-                <img src="icons/Twitter.png" alt="Twitter Icon">
-                <a href="#">
-                    <!-- <i class="fa fa-twitter" aria-hidden="true"></i> -->
-                </a>
+            <li class="github">
+                <img src="icons/Git.png" alt="GitHub Icon">
             </li>
             <li class="linkedin">
                 <img src="icons/Linkedin.png" alt="LinkedIn Icon">
-                <a href="#">
-                    <!-- <i class="fa fa-linkedin" aria-hidden="true"></i> -->
-                </a>
             </li>
-            <li class="github">
-                <img src="icons/Git.png" alt="GitHub Icon">
-                <a href="#">
-                    <!-- <i class="fa fa-github" aria-hidden="true"></i> -->
-                </a>
+            <li class="angel">
+                <img src="icons/AngelList.png" alt="AngelList Icon">
+            </li>
+            <li class="twitter">
+                <img src="icons/Twitter.png" alt="Twitter Icon">
             </li>
             <li class="medium">
                 <img src="icons/Medium.png" alt="Medium Icon">
-                <a href="#">
-                    <!-- <i class="fa fa-medium" aria-hidden="true"></i> -->
-                </a>
+            </li>
+        </ul>
+    </section>
+
+    <section class="work">
```

Comment:
- [ ] to match the Figma design, kindly add some space between the headline & the work section 🙏 

![image](https://user-images.githubusercontent.com/89282221/221351298-a23add26-7b4e-41a7-a09d-e007ef927c46.png)

Reply:
Hello, @ichala  I have implemented the feedback you gave me, can you take a look again, please? I'm awaiting your feedback once more.